tests/settings/settings_wallet/test_wallet_settings_saved_addresses_add.py

```diff
@@ -35,13 +35,13 @@ def test_wallet_settings_add_saved_address(main_screen: MainWindow, address: str
     with step('Fill in the form and preferred networks and add the address'):
         add_saved_address_popup.add_saved_address(name, address)
 
+    with step('Verify recently added saved address is present in the list'):
```

Comment:
the reason i was checking the toast message before checking the list is that toast message disappears quickly so perhaps it is better to check the toast first which appears right away when you click the Add button and then verify the list of addresses, wdyt @Valentina1133 ?

Reply:
I changed it on purpose, because after some changes in the app the toast message appears not so quickly and it’s better to check first that the address appeared. You can check it in nightly runs. Unfortunately we cannot add wait for for toast message , so this is the only option I found. If you have more ideas let me know I will try it

Reply:
sure, thats perfect then :)